__tests__/dev_setup.test.bats

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bats
+
+@test "homebrew is installed" {
+  mac_only
+
+  run found_exe brew
+  [ "$status" -eq 0 ]
+}
+
+@test "watchman is installed" {
+  mac_only
+
+  run found_exe watchman
+  [ "$status" -eq 0 ]
+}
+
+@test "yarn is installed" {
+  run found_exe yarn
+  [ "$status" -eq 0 ]
+}
+
+@test "react-native is installed" {
+  run found_exe react-native
+  [ "$status" -eq 0 ]
+}
+
+@test "node is installed" {
+  run found_exe node
+  [ "$status" -eq 0 ]
+}
+
+@test "java is installed" {
+  run found_exe java
+  [ "$status" -eq 0 ]
+}
+
+@test "android-studio is installed" {
+  linux_only
+
+  run found_exe android-studio
+  [ "$status" -eq 0 ]
+}
+
+@test "~/.profile_mobileapp exists" {
```

Comment:
Not sure this is actually needed.  Technically this only happens if the user selected to have the dev_setup script install android studio and set up the environment vars.  Probably OK to not check this I think.

Reply:
In non-interactive mode `-ni` the file is always made, so I think a test for it is warranted, unless we decide to conditionally write the file.